### ISSUE TITLE
Details about the German dataset training

### ISSUE THREAD

**Issue by robertjyh** [JinhuiYE/SignCL#2]
Hi, I ran your German dataset fine-tuning code "train_dsl_SignCL.py" on 4 V100s, but did not achieve the same results as your paper, especially the B@1 metric.
Could you provide the log of the German training and the training details about the parameters?

Also, I noticed that in the paper, you set λ to 0.01. But in the code, you set it to 0.06 along with warmup epochs. Does this have a big impact on the results?

**Comment by JinhuiYE** [comment]
hi, thanks for your reading out. 

For results on the German dataset, refer to the discussion on the GitHub issue https://github.com/zhoubenjia/GFSLT-VLP/issues/10#issuecomment-1953821489. I guess this is the main potential issue.
Seems the counter of GPU is most related to the performance. You may need to adapter the lr when using different GPUs.

Regarding training parameters, the value of λ had a significant impact on the results as highlighted in the paper. However, the number of warmup epochs and decay did not show a substantial effect. (0.01 is what we used, sorry for the confusion)

The CSL-Daily dataset was primarily used as an example, but the parameters were consistent across different datasets without any special design for specific datasets. You can refer to the train_csl_SignCL.py script for more details on the training process.

I will try to re-run on my side and get back to you with log

**Comment by robertjyh** [comment]
I will try to retrain according to what you said. Thanks a lot!
